lib/polyfill/mediasource.js

```diff
@@ -36,10 +36,12 @@ shaka.polyfill.MediaSource = class {
       // Chromecast cannot make accurate determinations via isTypeSupported.
       shaka.polyfill.MediaSource.patchCastIsTypeSupported_();
     } else if (safariVersion) {
-      // TS content is broken on Safari in general.
-      // See https://github.com/google/shaka-player/issues/743
-      // and https://bugs.webkit.org/show_bug.cgi?id=165342
-      shaka.polyfill.MediaSource.rejectTsContent_();
+      // TS content is broken on Safari in 13 or less.
+      if (safariVersion <= 13) {
+        // See https://github.com/google/shaka-player/issues/743
+        // and https://bugs.webkit.org/show_bug.cgi?id=165342
```

Comment:
My only concern with this PR is that this WebKit issue was never closed by Apple.  Can you please confirm that the repro steps in the bug no longer reproduce the issue in Safari 14?

1. Navigate to http://storage.googleapis.com/shaka-demo-assets/_bugs/safari-10-mse-abort/index.html
2. Open the JS console
3. Look for an error message such as "Segment size should be 10" or "The index is not in the allowed range"

Reply:
@joeyparrish : IndexSizeError: The index is not in the allowed range.

<img width="1376" alt="Captura de pantalla 2020-12-23 a las 20 54 47" src="https://user-images.githubusercontent.com/8983024/103032758-216f0400-4561-11eb-9694-7c0eba8a37c2.png">

Reply:
Now that I look at the test code ... It is for MP4 and for sourceBuffer.abort here the content of TS is not taken into account.

Note: this only enables TS the rest of the polyfill is not touched.

Reply:
Ah, interesting.  I wonder if we have the wrong bug link here.  Let me do some re-testing with TS content in Safari 13 after the holiday, and we'll come back to this.

Reply:
@joeyparrish Have you been able to re-test it?

Reply:
I tested in Safari 13.1.3, and it is still broken.

Here's my methodology:

1. Modify the MSE polyfill not to reject TS content in Safari
2. Load the demo app
3. Uncheck the "Use native HLS on Safari" option
4. Play "Art of Motion (HLS, TS)"
5. In the JS console, confirm that we're using MSE and not native HLS: `video.ui.getControls().getLocalPlayer().mediaSourceEngine_` should be non-null
6. In the JS console, confirm that we're not transmuxing: `video.ui.getControls().getLocalPlayer().mediaSourceEngine_.transmuxers_.video` should be undefined
7. Force a change of resolution in the UI

In Safari 13, playback completely breaks when I change resolution.  Changing resolution triggers an abort() call in MSE, which is meant to reset the pipeline, but which causes subsequent appends of TS content to fail (silently) in Safari.

Reply:
If these steps work correctly in Safari 14, I'll approve this change.  I don't have Safari 14 yet, but I'm coordinating OS updates in our lab with the rest of the team.

Reply:
I have tried on two different Macs, and I have conflicting results.

In one of them the change between 720 and 1080 works correctly, but in the rest of the resolutions it does not, on the other hand in the other mac the small resolutions work, but the high ones do not.

Because of this TS is not functional yet, so I close this PR. :(